app/src/main/java/com/work/restaurant/view/diary/add_eat/AddEatFragment.kt

```diff
@@ -113,14 +133,20 @@ class AddEatFragment : BaseFragment(R.layout.diary_add_eat),
             SimpleDateFormat("yyyy년 MM월 dd일", Locale.getDefault()).format(currentTime)
 
 
-        val dateText1 = SimpleDateFormat("yyyyMMdd", Locale.getDefault()).format(currentTime)
+        val dateText1 = SimpleDateFormat("yyyyMMddahhmm", Locale.getDefault()).format(currentTime)
```

Comment:
```suggestion
        val dateText1 = SimpleDateFormat("yyyy-MM-dd-ahhmm", Locale.getDefault()).format(currentTime)
        val array = dateText1.split["-"]
```
split을 사용하는 것이 substring 보다 좀 더 의미가 명확하지 않을까요?

Reply:
9c25d35 수정하였습니다. 
그리고 
> "${dateArray[0]}년 ${dateArray[1]}월 ${dateArray[2]}일"
이런식으로 한거를 SuppressLint하지 않고 
![image](https://user-images.githubusercontent.com/54328309/72970450-094fab80-3e0b-11ea-909c-e93e71eadf6b.png)
이러한 이유로 변경해 보았습니다.

Reply:
그런데 이제보니까 그냥 SimpleDateFormat에서
yyyy년 MM월 dd일 이렇게 포맷을 사용하면 되지않을까요?

Reply:
음.. 그러면 저런거 할 필요 없겟네요?ㅋㅋㅋㅋ 근데 main에서는 위에같이 하긴했어요ㅋㅋㅋ